SerialPrograms/Source/PokemonLZA/Programs/PokemonLZA_StallBuyer.cpp

```diff
@@ -47,24 +47,25 @@ class BerryBuyer_Descriptor::Stats : public StatsTracker{
     std::atomic<uint64_t>& purchases;
     std::atomic<uint64_t>& errors;
 };
-std::unique_ptr<StatsTracker> BerryBuyer_Descriptor::make_stats() const{
+std::unique_ptr<StatsTracker> StallBuyer_Descriptor::make_stats() const{
     return std::unique_ptr<StatsTracker>(new Stats());
 }
 
 
-BerryBuyer::BerryBuyer()
-    : BERRY_TYPE(
-        "<b>Berry to Purchase:",
+StallBuyer::StallBuyer()
+    : ITEM_POSITION(
+        "<b>Item Position to Purchase:",
         {
-            {BerryType::POMEG,  "pomeg",  "Pomeg (HP)"},
-            {BerryType::KELPSY, "kelpsy", "Kelpsy (Attack)"},
-            {BerryType::QUALOT, "qualot", "Qualot (Defence)"},
-            {BerryType::HONDEW, "hondew", "Hondew (Special Attack)"},
-            {BerryType::GREPA,  "grepa",  "Grepa (Special Defence)"},
-            {BerryType::TAMATO, "tamato", "Tamato (Speed)"},
+            {ItemPosition::FirstFromTop,  "FirstFromTop",  "First From Top"},

```

Comment:
If you're gonna do this, let's use numbers instead.

Item Slot:
1 (top)
2 (2nd from top)
3 (...)
...
7 (bottom)